### DISCUSSION TITLE
docs clarification request?

### DISCUSSION THREAD

**Discussion by MegMM** [noDRM/DeDRM_tools#174] [Ideas]
Hello,
The Github site does not have a clear link to the correct download ZIP file. A simple link in the top-level doc would avoid frustration or confusion. On the top-level page, https://github.com/noDRM/DeDRM_tools, clearly point to the ZIP on https://github.com/noDRM/DeDRM_tools/releases

While https://github.com/noDRM/DeDRM_tools/discussions/112 addresses the issue, it would be easy and much less frustrating to resolve with a simple clarification in the top README. Just a thought...
Thank you for managing this exceedingly useful plugin!
-Meghan

**Comment by ElleKayEm** [comment]
There is a link to the latest release on the main page.
